.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,31 +1,67 @@
+<!--
+* New contributors are highly encouraged to read our
+  [CONTRIBUTING](/CONTRIBUTING.md) documentation.
+* The pull request:
+  * Should only fix one issue or add one feature at a time.
+  * Must update the test suite for the relevant functionality.
+  * Should pass all status checks before being reviewed or merged.
+* Commit titles should be prefixed with general area of pull request's change.
+* Draft PRs should be prefixed with `[WIP]` in their title.
+
+-->
 ### What does this PR do?
 
-A brief description of the change being made with this pull request.
+<!--
+* A brief description of the change being made with this pull request.
+* If the description here cannot be expressed in a succint form, consider
+  opening multiple pull requests instead of a single one.
+-->
 
 ### Motivation
 
-What inspired you to submit this pull request?
+<!--
+* What inspired you to submit this pull request?
+* Link any related GitHub issues here.
```

Comment:
```suggestion
* Link any related GitHub issues or PRs here.
```
I think sometimes we miss links between PRs (eg: follow-up PRs that fix edge cases from previous PRs).